components/Paragraph.tsx

```diff
@@ -10,7 +10,7 @@ export const Paragraph = ({ children, ...props }: ParagraphProps) => {
       sx={{
         mt: Spacing['16px'],
         mb: Spacing['24px'],
-        '&:first-child': { mt: 0 },
+        '&:first-of-type': { mt: 0 },
```

Comment:
![image](https://user-images.githubusercontent.com/7361780/215563515-3bb46854-a1ed-49f4-862f-f7534320b969.png)

Reply:
Hmm, these are not exactly equivalent, but I just tested and it doesn't break anything... nor does removing this entire rule... or the following one (the `&:last-child: { mb: 0 }` one). So, I propose doing that. 😄